src/index.js

```diff
@@ -0,0 +1,160 @@
+        displayTemp.textContent = temperature;
+        updateTempColor(temperature);
+        updateLandscape(temperature);
+        landscape.textContent = lands;
+    }
+    const updateTempColor = (temperature) => {
+        if (temperature >= 80) {
+        displayTemp.style.color = "red";
+        } else if (temperature >= 70 && temperature <= 79) {
+        displayTemp.style.color = "orange";
+        } else if (temperature >= 60 && temperature <= 69) {
+        displayTemp.style.color = "yellow";
+        } else if (temperature >= 50 && temperature <= 59) {
+        displayTemp.style.color = "green";
+        } else {
+        displayTemp.style.color = "teal";
+        }
+    }
+    const updateLandscape = (temperature) => {
+        if (temperature >= 80) {
+        lands = "🌵   🐍 🦂 🌵🌵  🐍 🏜 🦂";
+        } else if (temperature >= 70 && temperature <= 79) {
+        lands = "🌸🌿🌼  🌷🌻🌿 ☘️🌱 🌻🌷";
+        } else if (temperature >= 60 && temperature <= 69) {
+        lands = "🌾🌾   🍃 🪨 🛤 🌾🌾🌾 🍃";
+        } else if (temperature <= 59) {
+        lands = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+        }
+    }
+    const increaseTemperature = () => {
+        temperature++;
+        updateTemp(temperature);
+    }
+    const decreaseTemperature = () => {
+        temperature--;
+        updateTemp(temperature);
+    }  
+
+    increaseTemp.addEventListener("click", increaseTemperature);
+    decreaseTemp.addEventListener("click", decreaseTemperature);
+    updateTemp(temperature);
+    
+    // ------------- wave 3 naming the city ------------------
+    // Realtime Text City info will come from input value of cityInput
+    const updateCityName = () => {
+        cityName.textContent = cityInput.value;
+    }
+    
+    // event listener to listen for input
+    cityInput.addEventListener("input", updateCityName);
+    
+    // ------------- wave 5 selecting sky ------------------
+    result.textContent = "🌦 🌈  ☁️☁️☁️  ❄️ 🌨 ☁️"
+    selectSky.addEventListener("change", (event) => {
+        let newSky = event.target.value
+        if (newSky == 'sunny') {
+            result.textContent = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️"
+        } else if (newSky == 'cloudy'){
+            result.textContent = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️"
+        } else if (newSky == 'rainy'){
+            result.textContent = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧"
+        } else if (newSky == 'snowy'){
+            result.textContent = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+        } else {
+            result.textContent = "🧨 🧨 🧨 🧨 🧨 🧨"
+        }
+    })
+    
+    // -------- wave 6    reset button event listener ---------
+    cityName.textContent = state.city
+    resetButton.addEventListener("click", function () {
+        cityInput.value = state.city; 
+        cityName.textContent = state.city;
+        updateTemp("65")
+    } )
+
+    // -------- wave 4    calls API  ---------
+    const findLatitudeAndLongitude = async () => {
+        // let latitude, longitude;
+        await axios.get('http://127.0.0.1:5000/location',
```

Comment:
I see that you used the `async/await` keywords in a few of your functions. I love that you are making use of them! Here's a tip that will allow you to bring your use more in line with the keywords intention. `async/await` is intended to be be used **in-place** of `promises`. So when using them your code should a little more like this:

```javascript
const findLatitudeAndLongitude = async () => {
        const resp = await axios.get('http://127.0.0.1:5000/location',{
            params: {
                q:cityInput.value,
                format: 'json'
            }})
        
        try {
            state.lat= response.data[0].lat;
            state.lon= response.data[0].lon;
          
            return {
            cityLat: state.lat,
            cityLon: state.lon
          }
          
        } catch {
          console.log('error in findLatitudeAndLongitude!')
        }
 }
```